### PR TITLE
Improve KycRequest statuses

### PR DESCRIPTION
**Pull request by justnero**
[Preview](https://preview.redoc.ly/openapi-full-api/kyc-request-statuses/tag/KYC-Documents#operation/GetKycRequest!c=200&path=status&t=response)

### CODE REVIEW

File: openapi/components/schemas/Common/CommonKycRequest.yaml [hunk 1]
```diff
@@ -37,11 +37,21 @@ properties:
...
-      - rejected
       - fulfilled
-      - expired
+      - failed
+      - incomplete
+      - in-progress
+    x-enumDescriptions:
+      abandoned: Request expired and has at least one document uploaded
+      expired: Request expired without any document uploaded
+      pending: All requested documents are provided and have status either `accepted`, `pending` or `in-progress`
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonKycRequest.yaml:50]:
```suggestion
      pending: All requested documents are provided and have the status `accepted`, `pending` or `in-progress`
```

File: openapi/components/schemas/Common/CommonKycRequest.yaml [hunk 2]
```diff
@@ -37,11 +37,21 @@ properties:
...
+      fulfilled: All requested documents are provided and `accepted`
+      failed: At least one requested document exhausted attempts and has no `accepted`, `pending` or `in-progress` document
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonKycRequest.yaml:52]:
```suggestion
      failed: At least one requested document exhausted all attempts and does not have the status `accepted`, `pending` or `in-progress`
```

File: openapi/components/schemas/Common/CommonKycRequest.yaml [hunk 3]
```diff
@@ -37,11 +37,21 @@ properties:
...
+      incomplete: At least one requested document is `accepted`, but not all of them
+      in-progress: No requested documents are `accepted` and not all are uploaded in `pending` or `in-progress` statuses
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonKycRequest.yaml:54]:
```suggestion
      in-progress: No requested documents are `accepted` and not all uploaded documents have the status `pending` or `in-progress`
```

File: openapi/components/schemas/Common/CommonKycRequest.yaml [hunk 4]
```diff
@@ -37,11 +37,21 @@ properties:
...
+      - failed
+      - incomplete
+      - in-progress
+    x-enumDescriptions:
+      abandoned: Request expired and has at least one document uploaded
+      expired: Request expired without any document uploaded
+      pending: All requested documents are provided and have status either `accepted`, `pending` or `in-progress`
+      fulfilled: All requested documents are provided and `accepted`
+      failed: At least one requested document exhausted attempts and has no `accepted`, `pending` or `in-progress` document
+      incomplete: At least one requested document is `accepted`, but not all of them
```
Comment (Boydbueno) [inline openapi/components/schemas/Common/CommonKycRequest.yaml:53]:
```suggestion
      incomplete: At least one requested document has the `accepted` status, but not all of them
```

**Review by Boydbueno — COMMENTED** [review]
Adjusted a few of the sentences. Mostly nitpicks

**Review by swilczynski — APPROVED** [review]

**Review by Boydbueno — APPROVED** [review]

**Pull request merged**